### ISSUE TITLE
[BUG] File got overwritten without notification when uploading another file with same filename

### ISSUE THREAD

**Issue by ghost** [tobychui/arozos#67]
**Describe the bug**
When uploading files via browser, it does not ask for anything before overwriting files with existing file with same filename. Usually file managers will ask for conformation first or auto renaming but is currently missing.

**To Reproduce**
1. Drag a file to the desktop for uploading
2. Drag another file with the same filename to upload
3. There will be two files with the same name on the desktop
4. Open both the files, they will both point to the same file uploaded in step 2

**Expected behavior**
Either (a) Renaming second file to screenshot-2.png automatically or (b) Prompt to choose file overwrite confirmation

**Screen Recording**
https://user-images.githubusercontent.com/68219227/123539605-29586c80-d76d-11eb-91db-f8c5f2053870.mp4

**Host Information(please complete the following information):**
 - OS: Debian
 - ArozOS version: v1.113
 - Hardware: Raspberry Pi 3B+

**Client Information(please complete the following information):**
 - OS: Windows 10
 - Browser: Firefox
 - Version: 89.0.2

**Comment by tobychui** [comment]
Hi @cakkomilk, I have checked and been able to reproduce your issue.

For duplicated desktop icon, it is a bug of the desktop rendering pipeline and we have scheduled a big update for the desktop upload mechanism in v1.115. I guess I will include the patch in that release instead of the next one (v1.114).

For the parts about overwriting confirmation or file automatically renaming, this should belongs to another issue (enhancement). I will start a discussion in our Telegram groups after v1.114 release and see which is a better way to implement this feature without breaking many things.

Thanks for the issue report :)
